Source/WebCore/Modules/notifications/Notification.h

```diff
@@ -88,7 +88,7 @@ class Notification final : public RefCounted<Notification>, public ActiveDOMObje
     const String& tag() const { return m_tag; }
     const URL& icon() const { return m_icon; }
     JSC::JSValue dataForBindings(JSC::JSGlobalObject&);
-    bool silent() const { return m_silent == std::nullopt ? false : *m_silent; }
+    const std::optional<bool> silent() const { return m_silent; }
```

Comment:
I'm surprised we don't use `Markable<>` for our IDL stuff. cc @mdubet

Reply:
Markable doesn't work with bool because we do need the 3 states (null, false, true) and we can't encode them on a single bit.

Reply:
The first `const` (on the return value) seems unnecessary: `std::optional<bool> silent() const { return m_silent; }`

Reply:
oops, always get those mixed up.